packages/schematics/src/command-line/workspace-schematic.ts

```diff
@@ -1,48 +1,23 @@
 import { execSync } from 'child_process';
 import * as fs from 'fs';
-import { readFileSync, statSync, writeFileSync } from 'fs';
+import { readFileSync, writeFileSync } from 'fs';
 import * as path from 'path';
 import { copySync, removeSync } from 'fs-extra';
-import {
-  FileSystemEngineHost,
-  NodeModulesEngineHost,
-  validateOptionsWithSchema
-} from '@angular-devkit/schematics/tools';
-import { BuiltinTaskExecutor } from '@angular-devkit/schematics/tasks/node';
-import {
-  CollectionDescription,
-  EngineHost,
-  RuleFactory,
-  Schematic,
-  SchematicDescription,
-  SchematicEngine,
-  Tree,
-  DryRunSink,
-  TypedSchematicContext,
-  HostSink,
-  HostTree
-} from '@angular-devkit/schematics';
-import { of } from 'rxjs';
-import { concat, concatMap, ignoreElements, map } from 'rxjs/operators';
-import { Url } from 'url';
+import { NodeWorkflow } from '@angular-devkit/schematics/tools';
+import { UnsuccessfulWorkflowExecution } from '@angular-devkit/schematics';
 
 import * as yargsParser from 'yargs-parser';
-import { CoreSchemaRegistry } from '@angular-devkit/core/src/json/schema';
-import { standardFormats } from '@angular-devkit/schematics/src/formats';
 import * as appRoot from 'app-root-path';
-import { virtualFs, normalize } from '@angular-devkit/core';
-import { NodeJsSyncHost } from '@angular-devkit/core/node';
+import { virtualFs, normalize, JsonObject } from '@angular-devkit/core';
+import { NodeJsSyncHost, createConsoleLogger } from '@angular-devkit/core/node';
 
 const rootDirectory = appRoot.path;
 
 export function workspaceSchematic(args: string[]) {
   const outDir = compileTools();
   const schematicName = args[0];
-  const { schematic, host, engine } = prepareExecutionContext(
-    outDir,
-    schematicName
-  );
-  executeSchematic(schematicName, parseOptions(args), schematic, host, engine);
+  const workflow = createWorkflow();
```

Comment:
For some reason, schematics which were used within other schematics started not receiving default values? 🤷‍♀️ To fix this, I copied more of what's happening in `@angular-devkit/schematics-cli`